### PR TITLE
SERVICE: Created CategoryForViewDTO,ICategoryService and CategoryService

### LINKED ISSUE #88: SERVICE: Create CategoryForViewDTO, ICategoryService and CategoryService

### PR DESCRIPTION
**Pull request by Dilshod-Abdijabbarov**
Closes #88

### CODE REVIEW

**Comment by ShohruhUzDev** [comment]
Issue tag

File: BelissimoCloneWPF.Service/Services/Categories/CategoryService.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+            return isDelete;
+        }
+
+        public async ValueTask<IEnumerable<CategoryForViewDTO>> GetAllAsync(PaginationParams @params, Expression<Func<Category, bool>> expression)
+        {
+
+            var categories = categoryRepository.GetAll(expression: expression, isTracking: false);
+
+
```
Comment (ShohruhUzDev) [inline BelissimoCloneWPF.Service/Services/Categories/CategoryService.cs:60]: Remove this line

File: BelissimoCloneWPF.Service/Services/Categories/CategoryService.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                throw new BelissimoCloneWPFException(404, "Category Not found");
+
+            await categoryRepository.SaveChangesAsync();
+
+            return isDelete;
+        }
+
+        public async ValueTask<IEnumerable<CategoryForViewDTO>> GetAllAsync(PaginationParams @params, Expression<Func<Category, bool>> expression)
+        {
+
```
Comment (ShohruhUzDev) [inline BelissimoCloneWPF.Service/Services/Categories/CategoryService.cs:57]: remove line

File: BelissimoCloneWPF.Service/Services/Categories/CategoryService.cs (hunk 1 above)
Comment (MuhammadKasimov) [inline BelissimoCloneWPF.Service/Services/Categories/CategoryService.cs:60]: remove

**Review by MuhammadKasimov — CHANGES_REQUESTED** [review]

File: BelissimoCloneWPF.Service/Services/Categories/CategoryService.cs (hunk 1 above)
Comment (Dilshod-Abdijabbarov) [reply BelissimoCloneWPF.Service/Services/Categories/CategoryService.cs]: Remove line

**Pull request merged**